portfolio/src/main/webapp/script.js

```diff
@@ -51,11 +51,18 @@ async function getMessage() {
  * Gets the current messages and displays them as comments.
  */
 async function getCurrentMessages() {
+
+
```

Comment:
Nit: Remove blank lines at beginning of function

Reply:
Done!